var/SymfonyRequirements.php

```diff
@@ -518,7 +518,7 @@ function_exists('simplexml_import_dom'),
         if (extension_loaded('suhosin')) {
             $this->addPhpIniRequirement(
                 'suhosin.executor.include.whitelist',
-                function ($cfgValue) { return false !== stripos($cfgValue, "phar"); },
+                function ($cfgValue) { return false !== stripos($cfgValue, 'phar'); },
```

Comment:
Note this file may loose these changes at the first composer update.

Reply:
Yes, I understand.

Reply:
You must revert changes here :/

Reply:
Okay, I will.

Reply:
Fixed